src/Microsoft.DotNet.ProjectJsonMigration/Rules/TemporaryMutateProjectJsonRule.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectModel;
+using System.IO;
+using Newtonsoft.Json.Linq;
+using System.Text;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    /// <summary>
+    /// This rule is temporary while project.json still exists in the new project system.
+    /// It renames your existing project.json (if output directory is the current project directory),
+    /// creates a copy, then mutates that copy.
+    /// 
+    /// Mutations:
+    ///  - inject a dependency on the Microsoft.SDK targets
+    ///  - removing the "runtimes" node.
+    /// </summary>
+    public class TemporaryMutateProjectJsonRule : IMigrationRule
+    {
+        private static string s_sdkPackageName => "Microsoft.DotNet.Core.Sdk";
```

Comment:
This value is hard-coded here and in ProjectJsonParser.  Can we have a single place for it?  I know it will be changing soon.